releasenotes/notes/flare-add-performance-profiling-bbae32a7ca98eadc.yaml

```diff
@@ -0,0 +1,12 @@
+# Each section from every releasenote are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+enhancements:
+  - |
+    Changes flare creation so that archiving is handled from the flare process, leaving the agent to
```

Comment:
IMO, enhancements should only describe the new feature (a flare contains profiling data) and no the technical details.

Reply:
Makes sense.